cmd/mothership/mothership/start/http.go

```diff
@@ -128,6 +128,7 @@ func startWebserver(ctx context.Context, o *Options) error {
 	metrics.RegisterOccupancy(o.Registry.OccupancyRepository(), o.Config.Scheduler.Reconcilers, o.Logger())
 	metrics.RegisterProcessingDuration(o.Registry.ReconciliationRepository(), o.Logger())
 	metrics.RegisterWaitingAndNotReadyReconciliations(o.Registry.Inventory(), o.Logger())
+	metrics.RegisterDbPool(o.Registry.Inventory(), o.Logger())
```

Comment:
QQ.. why are we using the inventory as source for the DBStats? All repositories are sharing the same sql.Db instance (=Pool) as they are using all the same conneciton (see https://github.com/kyma-incubator/reconciler/blob/main/internal/persistency/registry.go#L33) - shouldn't we add this `Stats()` function to the DB-connection interface ?

Reply:
Quick question, the code you send me shows that a new repository is also initializing a new DB Pool due to `conn, err := cf.NewConnection()` no? nevertheless, we can surely move the logic to the connection layer, i for sure agree